pages/activity/[id].js

```diff
@@ -23,27 +25,34 @@ export default function SingleActivity({ editTag }) {
 
         <li>
           <TagList
+            tags={activity.tags}
             activityId={activity.id}
             tagId={activity.tags}
             editTag={editTag}
           />
         </li>
 
-        {activity.imageURL && <TagImg src={activity.imageURL} />}
-        <li>Date: {activity.date}</li>
+        {activity.imageURL && <Img src={activity.imageURL} />}
+        <li>{activity.date}</li>
         <li>
           {activity.url && (
             <Link href={activity.url} target="_blank">
               your URL
             </Link>
           )}
         </li>
+        <div className="buttonlink">
```

Comment:
Another CSS class. `buttonlink` as name is also quite confusing as these stylings apply to the div. Maybe `Wrapper` or `Container` or `ActionContainer` is better?